nativelib/src/main/resources/scala-native/scala-native.properties

```
@@ -0,0 +1,13 @@
+# output via debugging
+project.groupId = org.scala-native
+project.artifactId = nativelib
+# setup define based on GC selected
+project.gcProject = true
+
+# CG define
+preprocessor.defines = NDEBUG
+
+# path to vendored libunwind
+compile.include.path = platform, posix, libunwind
```

How about making all paths numbered? This way it would more consistient, 
```suggestion
compile.include.path0 = platform, posix, libunwind
```

---

I think we can just go with the previous suggestion - I'll try and update the PR and docs tonight.